### ISSUE TITLE
UptimeKuma v2 support

### ISSUE THREAD

**Issue by BigBoot** [BigBoot/AutoKuma#99]

**Comment by barcar** [comment]
This works perfectly. Thank you for implementing it so quickly.

**Comment by undaunt** [comment]
One thing I noticed with my current config is when I did a fresh compose up -d on uptime kuma itself, autokuma created a second copy of the monitors I currently had instead of detecting them. They were created both times by Autokuma.

![image](https://github.com/user-attachments/assets/c36c5d06-a3cd-4c77-86b6-d77bee14029c)

I've also noted a lot of this:

```

WARN [kuma_client::util] The server rejected the login: Too frequently, try again later.
WARN [kuma_client::util] Error while handling 'Info' event: The server rejected the login: Too frequently, try again later.
WARN [kuma_client::client] Timeout while waiting for Kuma to get ready...
WARN [autokuma::sync] Encountered error during sync: It looks like the server is expecting a username/password, but none was provided
WARN [kuma_client::util] The server rejected the login: Too frequently, try again later.
WARN [kuma_client::util] Error while handling 'Info' event: The server rejected the login: Too frequently, try again later.
WARN [kuma_client::client] Timeout while waiting for Kuma to get ready...
WARN [autokuma::sync] Encountered error during sync: It looks like the server is expecting a username/password, but none was provided
WARN [kuma_client::util] The server rejected the login: Too frequently, try again later.
WARN [kuma_client::util] Error while handling 'Info' event: The server rejected the login: Too frequently, try again later.
WARN [kuma_client::client] Timeout while waiting for Kuma to get ready...
WARN [autokuma::sync] Encountered error during sync: It looks like the server is expecting a username/password, but none was provided
WARN [kuma_client::util] The server rejected the login: Too frequently, try again later.
WARN [kuma_client::util] Error while handling 'Info' event: The server rejected the login: Too frequently, try again later.
WARN [kuma_client::client] Timeout while waiting for Kuma to get ready...
WARN [autokuma::sync] Encountered error during sync: It looks like the server is expecting a username/password, but none was provided
WARN [kuma_client::util] The server rejected the login: Too frequently, try again later.
WARN [kuma_client::util] Error while handling 'Info' event: The server rejected the login: Too frequently, try again later.
WARN [kuma_client::client] Timeout while waiting for Kuma to get ready...
WARN [autokuma::sync] Encountered error during sync: It looks like the server is expecting a username/password, but none was pro
```

**Comment by undaunt** [comment]
Thanks, I just bumped up the sync interval.

Re: the first point, this is a net new test setup with only those containers. I just brought the stacks up, down, then up again. There isn't an easy migration path from v1 to v2 with current historical data, they're basically stating due to bandwidth reasons they won't officially support it, but others have posted info on how to create a mysql database and populate it with converted sqlite data via an export.

**Comment by bnctth** [comment]
Hi!
The latest update seems to have broken autokuma for ukv2. I recreated my uptimekuma server but ak can't create any monitors. It throws out the following logs repeatedly:
```
autokuma-1  | WARN [kuma_client::util] [backups-nasty.toml] No monitor named backups could be found
autokuma-1  | WARN [kuma_client::util] [backups-nasty-other_devices.toml] No monitor named backups-nasty could be found
autokuma-1  | WARN [kuma_client::util] [backups-vivo.toml] No monitor named backups could be found
autokuma-1  | WARN [kuma_client::util] [backups-nasty-self.toml] No monitor named backups-nasty could be found
autokuma-1  | WARN [kuma_client::util] [backups-asus_viki.toml] No monitor named backups could be found
autokuma-1  | WARN [kuma_client::util] [backups-nasty-db.toml] No monitor named backups-nasty could be found
autokuma-1  | WARN [autokuma::entity] Cannot create monitor uptime-kuma because referenced monitor with services is not found
autokuma-1  | WARN [kuma_client::util] Error while parsing uptime-kuma-auto-kuma: data did not match any variant of untagged enum EntityWrapper!
autokuma-1  | WARN [autokuma::sync] Encountered error during sync: Error while trying to parse labels: data did not match any variant of untagged enum EntityWrapper
```
The referenced groups are supposed to be created  by ak too. For the enum problem I haven't had time to actually debug the code, but the latest uk update did have a pr merged for monitor tags: https://github.com/louislam/uptime-kuma/pull/5298
I didn't change any of the already working monitor definitions and am running the images `louislam/uptime-kuma:beta` (`sha256:752118f891ea991180124e3fc7edbc1865a58cb03e15e612ecbc68065b1d4b9f`) and `ghcr.io/bigboot/autokuma:uptime-kuma-v2-master` (`sha256:74bccf145554cce2acf63676d4b98fafdf1e710e60150733fcac8b5b1c364301`)

Thanks for the help and all the good work you do!

**Comment by BigBoot** [comment]
Hi @bnctth, I don't think there's any breaking change, this looks rather like a problem with your labels, let's try to break this down.

```
autokuma-1  | WARN [kuma_client::util] [backups-nasty.toml] No monitor named backups could be found
autokuma-1  | WARN [kuma_client::util] [backups-nasty-other_devices.toml] No monitor named backups-nasty could be found
autokuma-1  | WARN [kuma_client::util] [backups-vivo.toml] No monitor named backups could be found
autokuma-1  | WARN [kuma_client::util] [backups-nasty-self.toml] No monitor named backups-nasty could be found
autokuma-1  | WARN [kuma_client::util] [backups-asus_viki.toml] No monitor named backups could be found
autokuma-1  | WARN [kuma_client::util] [backups-nasty-db.toml] No monitor named backups-nasty could be found
```

These are "expected" when creating nested setups, autokuma knows about these dependencies but they have not been created yet, as a result they are skipped till later.

```
autokuma-1  | WARN [autokuma::entity] Cannot create monitor uptime-kuma because referenced monitor with services is not found
```
this one says that you have a monitor referencing a parent monitor with the autokuma id "services", however there doesn't seem to exist any such monitor definition. 

```
autokuma-1  | WARN [kuma_client::util] Error while parsing uptime-kuma-auto-kuma: data did not match any variant of untagged enum EntityWrapper!
autokuma-1  | WARN [autokuma::sync] Encountered error during sync: Error while trying to parse labels: data did not match any variant of untagged enum EntityWrapper
```
This error unfortunately isn't as clear, but it basically means you have a definition (`uptime-kuma-auto-kuma`) with a missing or invalid "type".

**Comment by bnctth** [comment]
@BigBoot  thanks for your reply! Turn out I had a pretty trivial problem mixed with some red herring-y error messages: I had a typo in a snippet (`notificationIDList` instead of `notificationIdList˙, small d in Id), so I had the correct definition for the parents, but because of the error it never got to their definition.

**Comment by mlamoure** [comment]
Just noticed I'm getting this error over and over in my logs, not sure how long it's been going for.  Any ideas?

`
Invalid config: missing field `files`
`

**Comment by BigBoot** [comment]
Sorry I added the `files.follow_symlinks` option and forgot to set a default, should be working now

**Comment by mlamoure** [comment]
> Sorry I added the `files.follow_symlinks` option and forgot to set a default, should be working now

That's why it's a beta.  Nice work on this app.  My Uptime Kuma is nearly 100% config driven.  Two comments / questions why I have you:

1- I have multiple remote docker hosts.  I think I notice that if a host is down, AutoKuma gets "stuck" waiting for that node to come up, rather than ignoring and moving on.  If you add labels to a docker host that is up while another is down, the new labels won't get applied until all nodes are up again.

2- An ask of mine would be more verbose messaging where a mistake is made -- which docker host, which stack/service it's related to, etc.

**Comment by cronventis** [comment]
I'm currently testing the V2 CLI... It works fine for "Monitors"... but not for "Status Page" ... either I'm missing something, or it might be broken:

I tried multiple ways to "edit" a Status Page called alphaci. First I use `kuma statuspage get alphaci` this returns a json that I modify (add a monitor id & its name) and then store in a /tmp/statuspage.json

I tried:

`kuma status-page edit alphaci < /tmp/statuspage.json`
`cat /tmp/statuspage.json | kuma status-page edit alphaci`
`kuma status-page edit alphaci < /tmp/statuspage.json`

Error in all cases was `{"error":"No such file or directory (os error 2)"}`

Nothing worked the way I expected it.. Maybe I'm doing it wrong?

The json looks like this:

```json
{
    "customCSS": "body {\n  \n}\n",
    "domainNameList": [],
    "icon": "/icon.svg",
    "id": 2,
    "publicGroupList": [
        {
            "id": 2,
            "monitorList": [
                {
                    "id": 2,
                    "name": "alphaci",
                    "type": "group"
                },
                {
                    "id": 79,
                    "name": "cloud-1155",
                    "type": "keyword"
                },
                {
                    "id": 206,
                    "name": "ONV005",
                    "type": "keyword"
                }
            ],
            "name": "Services",
            "weight": 1
        }
    ],
    "published": true,
    "showCertificateExpiry": true,
    "showPoweredBy": true,
    "showTags": false,
    "slug": "alphaci",
    "theme": "auto",
    "title": "alphaci"
}
```

It would be awesome to get a little insight on what I might be doing wrong.

**Comment by cronventis** [comment]
Bit is also have another issue... So I'm running this script during deployment of a new site (lets say there are quite a lot of them...)
This is running inside a docker, so for every site it will run its own.. I can have them run in parallel to be faster, but then I will get a problem with the RATE-LIMIT buy uptime kuma... So I was wondering: does the CLI use one connection? Or does it create a connection for every request described here?

![Image](https://github.com/user-attachments/assets/12128907-1761-4d20-8300-2e1fb5a09051)

**Comment by chupacabra71** [comment]
I'm most likely doing something wrong on my end with uptime-kuma v2.  i see this in the autokuma log:

```
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    |                             _           _  __                         
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    |               /\           | |         | |/ /                         
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    |              /  \    _   _ | |_   ___  | ' /  _   _  _ __ ___    __ _ 
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    |             / /\ \  | | | || __| / _ \ |  <  | | | || '_ ` _ \  / _` |
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    |            / ____ \ | |_| || |_ | (_) || . \ | |_| || | | | | || (_| |
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    |           /_/    \_\ \__,_| \__| \___/ |_|\_\ \__,_||_| |_| |_| \__,_|  
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    |                                                        v0.8.0-3236fb93
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | ERROR [kuma_client::util] Error during connect
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [kuma_client::client] Timeout while waiting for Kuma to get ready...
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [autokuma::sync] Encountered error during sync: Timeout while trying to connect to Uptime Kuma server
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [kuma_client::util] Timeout while trying to call 'login'.
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [kuma_client::util] Error while handling 'Info' event: Timeout while trying to call 'login'.
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [kuma_client::util] Timeout while trying to call 'login'.
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [kuma_client::util] Error while handling 'Info' event: Timeout while trying to call 'login'.
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [autokuma::entity] Cannot create monitor mealie because referenced monitor with name group-mealie is not found
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [autokuma::entity] Cannot create monitor mealie-ext because referenced monitor with name group-mealie is not found
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | INFO [autokuma::sync] Creating new Monitor: group-mealie
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [kuma_client::util] Timeout while trying to call 'login'.
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [kuma_client::util] Error while handling 'Info' event: Timeout while trying to call 'login'.
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [kuma_client::util] Timeout while trying to call 'login'.
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [kuma_client::util] Error while handling 'Info' event: Timeout while trying to call 'login'.
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [autokuma::sync] Encountered error during sync: Timeout while trying to call 'editMonitor'.
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [kuma_client::util] Timeout while trying to call 'login'.
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [kuma_client::util] Error while handling 'Info' event: Timeout while trying to call 'login'.
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [kuma_client::util] Timeout while trying to call 'login'.
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [kuma_client::util] Error while handling 'Info' event: Timeout while trying to call 'login'.
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [autokuma::entity] Cannot create monitor mealie because referenced monitor with name group-mealie is not found
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [autokuma::entity] Cannot create monitor mealie-ext because referenced monitor with name group-mealie is not found
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | INFO [autokuma::sync] Creating new Monitor: group-mealie
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [kuma_client::util] Timeout while trying to call 'login'.
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [kuma_client::util] Error while handling 'Info' event: Timeout while trying to call 'login'.
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [autokuma::sync] Encountered error during sync: Timeout while trying to call 'add'.
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [kuma_client::util] Timeout while trying to call 'login'.
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [kuma_client::util] Error while handling 'Info' event: Timeout while trying to call 'login'.
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [kuma_client::util] Timeout while trying to call 'login'.
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [kuma_client::util] Error while handling 'Info' event: Timeout while trying to call 'login'.
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [kuma_client::util] Timeout while trying to call 'login'.
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [kuma_client::util] Error while handling 'Info' event: Timeout while trying to call 'login'.
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [autokuma::entity] Cannot create monitor mealie because referenced monitor with name group-mealie is not found
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [autokuma::entity] Cannot create monitor mealie-ext because referenced monitor with name group-mealie is not found
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | INFO [autokuma::sync] Creating new Monitor: group-mealie
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [kuma_client::util] Timeout while trying to call 'login'.
uptime-kuma_autokuma.1.tghrnk44yzdbkmxt0072jl0te@swarm-mgr01    | WARN [kuma_client::util] Error while handling 'Info' event: Timeout while trying to call 'login'.
```

The Uptime-Kuma compose is relatively simple:
```
---
version: '3.9'

services:
  uptime-kuma:
    image: louislam/uptime-kuma:beta
    deploy:
      replicas: 1
      restart_policy:
        condition: on-failure
        delay: 2s
        max_attempts: 5
      labels:
        # Enable Traefik for this service
        - "traefik.enable=true"
        ## HTTP Routers - internal
        - "traefik.http.routers.rtr-uptime-kuma.rule=Host(`monitor.${PRIMARY_DOMAIN}`)"
        - "traefik.http.routers.rtr-uptime-kuma.entrypoints=websecure"
        - "traefik.http.routers.rtr-uptime-kuma.service=svc-uptime-kuma"
        - "traefik.http.routers.rtr-uptime-kuma.priority=10"
        ## Middlewares
        - "traefik.http.routers.rtr-uptime-kuma.middlewares=global-forwardauth-authentik@file"
        ## HTTP Services 
        - "traefik.http.services.svc-uptime-kuma.loadbalancer.server.port=3001"
        ## UPTIME KUMA Labels
        # Kuma groups for organization
        - "KUMA.group-infrastructure.group.name=Infrastructure"
        - "KUMA.group-applications.group.name=Applications"

        - "KUMA.group-uptime-kuma.group.parent_name=group-infrastructure"
        - "KUMA.group-uptime-kuma.group.name=Uptime-Kuma"
        - "KUMA.uptime-kuma.http.parent_name=group-uptime-kuma"
        - "KUMA.uptime-kuma.http.name=Uptime Kuma Internal"
        - "KUMA.uptime-kuma.http.url=http://uptime-kuma:3001"
        - "KUMA.uptime-kuma-ext.http.parent_name=group-uptime-kuma"
        - "KUMA.uptime-kuma-ext.http.name=Uptime Kuma External"
        - "KUMA.uptime-kuma-ext.http.url=https://monitor.${PRIMARY_DOMAIN}"
    volumes:
      - data:/app/data
      - /var/run/docker.sock:/var/run/docker.sock:ro
    networks:
      - kuma-net
      - traefik-external

  autokuma:
    # image: ghcr.io/bigboot/autokuma:master
    image: ghcr.io/bigboot/autokuma:uptime-kuma-v2-master
    deploy:
      replicas: 1
      # mode: global
      restart_policy:
        condition: on-failure
        delay: 2s
        max_attempts: 5
      resources:
        limits:
          cpus: "0.5"
          memory: "256M"
        reservations:
          cpus: "0.25"
          memory: "128M"
      labels:
        - "traefik.enable=false"
    depends_on:
      - uptime-kuma        
    env_file: 
      - .env
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
      - autokuma-data:/data
    networks:
      - kuma-net

networks:
  traefik-external:
    external: true
  kuma-net:
    driver: overlay
    attachable: true  
```

and so is the Mealie one:
```
---
version: '3.9'

services:
  mealie:
    image: ghcr.io/mealie-recipes/mealie:v2.7.1 
    deploy:
      replicas: 1
      restart_policy:
        condition: on-failure
        delay: 2s
        max_attempts: 5
      labels:
        # Enable Traefik for this service
        - "traefik.enable=true"
        ## HTTP Routers - internal
        - "traefik.http.routers.rtr-mealie.rule=Host(`recipes.${PRIMARY_DOMAIN}`)"
        - "traefik.http.routers.rtr-mealie.entrypoints=websecure"
        - "traefik.http.routers.rtr-mealie.service=svc-mealie"
        - "traefik.http.routers.rtr-mealie.priority=10"
        ## Middlewares
        - "traefik.http.routers.rtr-mealie.middlewares=global-forwardauth-authentik@file"
        ## HTTP Services 
        - "traefik.http.services.svc-mealie.loadbalancer.server.port=9000"
        ## UPTIME KUMA Labels
        # - "KUMA.group-applications.group.name=Applications"
        - "KUMA.group-mealie.group.parent_name=group-applications"
        - "KUMA.group-mealie.group.name=Mealie"
        - "KUMA.mealie.http.parent_name=group-mealie"
        - "KUMA.mealie.http.name=Mealie Internal"
        - "KUMA.mealie.http.url=http://mealie:9000"
        - "KUMA.mealie-ext.http.parent_name=group-mealie"
        - "KUMA.mealie-ext.http.name=Mealie External"
        - "KUMA.mealie-ext.http.url=https://recipes.${PRIMARY_DOMAIN}"
    volumes:
      - mealie-data:/app/data/
    env_file: 
      - .env
    depends_on:
      - postgres
    networks:
      - mealie-network
      - traefik-external
      - kuma-net
```

It creates the group (i see it in the UI), but never creates the app.  In the UI, it actually keeps creating the same group over and over.

![Image](https://github.com/user-attachments/assets/bce40dd2-915a-467d-930e-f977f8c37aea)

**Comment by rv10guy** [comment]
Is there plans to support API keys that can be generated from the Uptime Kuma 2.0 UI rather than username/password?

**Comment by BigBoot** [comment]
> Is there plans to support API keys that can be generated from the Uptime Kuma 2.0 UI rather than username/password?

There still is no support for API keys on the SocketIO api afaik, the api keys only work for pushing metrics not for managing monitors

**Comment by BigBoot** [comment]
This is expected, you can change the sync interval with the `AUTOKUMA__SYNC_INTERVAL`/`sync_interval` (sync interval in seconds) config.

**Comment by phyzical** [comment]
👍 i actually just stumbled across that new var above and can confirm it helps the load issues so i will just throttle going forward.


 it also just  duplicated all monitors, but im not sure what caused this. if it happens again i will report back as after blatting the db and starting again i only have a single set

**Comment by phyzical** [comment]
Hmm is there some sort of statefile or something now that im not mapping? it seems when i do something that forces the container to be recreated it also recreates all monitors, but a simple reboot of the existing autokuma container doesn't cause this

<img width="766" height="897" alt="Image" src="https://github.com/user-attachments/assets/b05db626-bf66-41ba-8e76-d9c28f4cefc0" />

edit: ah yes opps i see theres a new volume `autokuma-data` (or maybe existing and i never noticed)

**Comment by radokristof** [comment]
I get the following error after upgrading uptime-kuma to v2 (on `latest` and `uptime-kuma-v2-master`):
```
thread 'tokio-runtime-worker' panicked at /usr/src/autokuma/kuma-client/src/client.rs:249:74:

called `Result::unwrap()` on an `Err` value: Error("unexpected trailing characters; the end of input was expected", line: 0, column: 0)

WARN [kuma_client::client] Timeout while waiting for Kuma to get ready...

WARN [autokuma::sync] Encountered error during sync: It looks like the server is expecting a username/password, but none was provided
```

Authentication is disabled in uptime-kuma.

**Comment by radokristof** [comment]
@BigBoot thank you for your help!

Relevant log lines:
```
2025-11-06 06:14:45.740 [kuma_client::client] TRACE: Client::on_any(Custom("maintenanceList"), Text([Object {"1": Object {"active": Bool(true), "cron": String(""), "dateRange": Array [String("2025-09-09 16:30:00"), String("2025-09-09 17:00:00")], "daysOfMonth": Array [], "description": String(""), "duration": Null, "durationMinutes": Number(0), "id": Number(1), "intervalDay": Number(1), "status": String("ended"), "strategy": String("single"), "timeRange": Array [Object {"hours": Number(0), "minutes": Number(0)}, Object {"hours": Number(0), "minutes": Number(0)}], "timeslotList": Array [Object {"endDate": String("2025-09-09 17:00:00"), "startDate": String("2025-09-09 16:30:00")}], "timezone": String("Europe/Budapest"), "timezoneOffset": String("+01:00"), "timezoneOption": String("Europe/Budapest"), "title": String("Migrating Marinero Survey"), "weekdays": Array []}, "2": Object {"active": Bool(true), "cron": String(""), "dateRange": Array [String("2025-10-05 23:35:00"), String("2025-10-06 00:10:00")], "daysOfMonth": Array [], "description": String(""), "duration": Null, "durationMinutes": Number(0), "id": Number(2), "intervalDay": Number(1), "status": String("ended"), "strategy": String("single"), "timeRange": Array [Object {"hours": Number(0), "minutes": Number(0)}, Object {"hours": Number(0), "minutes": Number(0)}], "timeslotList": Array [Object {"endDate": String("2025-10-06 00:10:00"), "startDate": String("2025-10-05 23:35:00")}], "timezone": String("UTC"), "timezoneOffset": String("+00:00"), "timezoneOption": String("UTC"), "title": String("Marina Manager Update"), "weekdays": Array []}, "3": Object {"active": Bool(true), "cron": String(""), "dateRange": Array [String("2025-10-08 16:40:00"), String("2025-10-08 17:00:00")], "daysOfMonth": Array [], "description": String(""), "duration": Null, "durationMinutes": Number(0), "id": Number(3), "intervalDay": Number(1), "status": String("ended"), "strategy": String("single"), "timeRange": Array [Object {"hours": Number(0), "minutes": Number(0)}, Object {"hours": Number(0), "minutes": Number(0)}], "timeslotList": Array [Object {"endDate": String("2025-10-08 17:00:00"), "startDate": String("2025-10-08 16:40:00")}], "timezone": String("UTC"), "timezoneOffset": String("+00:00"), "timezoneOption": String("UTC"), "title": String("Metabase Maintenance"), "weekdays": Array []}}]))
```
This is on current `master`

Edit: just for the sake of testing, after deleting the maintenance list, it started working so it is definitely something with the maintenance list...
